docs/docusaurus/docs/oss/oss.md

```diff
@@ -10,11 +11,13 @@ displayed_sidebar: gx_oss
 
 import LinkCardGrid from '@site/src/components/LinkCardGrid';
 import LinkCard from '@site/src/components/LinkCard';
+import OverviewCard from '@site/src/components/OverviewCard';
 
-<p class="DocItem__header-description">Use GX OSS to connect, test, and validate your Data Assets. If you're ready to get started see <a href='/docs/oss/guides/setup/get_started_lp'>Get started with GX OSS</a>.
-
- GX Cloud might be the solution if you don't want to manage your own configurations and infrastructure. <a href='https://greatexpectations.io/cloud'>Sign up here</a> to learn more and get early access to GX Cloud.
-</p>
+<OverviewCard title={frontMatter.title}>
+  Use GX OSS to connect, test, and validate your Data Assets.
```

Comment:
Same question here, do we want to say `Great Expectations (GX)` and then only use `GX`? cc. @kwcanuck @Rachel-Reverie

Reply:
@JessSaavedra I recommend using _GX_. We are now using the truncated form of the company everywhere, and we must assume that users know the long form of the company name.